molecule/Dockerfile-debian-java8.j2

```diff
@@ -40,4 +40,4 @@ RUN apt-get install --yes \
      confluent-security \
      confluent-control-center-fe \
      confluent-control-center \
-     confluent-schema-registry
+     confluent-schema-registry-plugins
```

Comment:
This is great! you should also add to this list:
https://github.com/confluentinc/cp-ansible/blob/7.0.1-post/roles/variables/defaults/main.yml#L525

The dockerfiles are for testing, but don't have impact on actual installs

Reply:
Thanks @domenicbove ! Just added

Reply:
The build seems to be disabled for community PRs (although I'm member of Confluent). Is there a way to mitigate this?

Reply:
@confluentinc/cp-ansible-dev I thought that community builds were fixed?

Reply:
Yeah, it's fixed. Built successfully on public jenkins - https://jenkins.public.confluent.io/job/cp-ansible/job/PR-894/4/
Surprisingly, it was also built on internal jenkins - https://jenkins.confluent.io/job/confluentinc-pr/job/cp-ansible/job/PR-894/4/ . This shouldn't have happened. Will check.

Reply:
@utkarsh5474 is that because I have to submit against 7.0.1-post instead of 7.0.x? For my changes to go into 7.0 release, which branch should the PR be against?

Reply:
Hey @Clarence97 
The changes should be submitted against 7.0.x to be made available in the next patch release.  
Public jenkins build should work for all PRs, regardless of the target branch. Did you see any discrepancy there?

Reply:
Since @Clarence97 is member of confluentinc, the public Jenkins job is not getting triggered here. For confluentinc member has to enable their role as public member here - https://github.com/orgs/confluentinc/people
<img width="1059" alt="Screenshot 2022-01-20 at 5 31 00 PM" src="https://user-images.githubusercontent.com/12181183/150335084-853e5d91-cd58-4570-b1ed-98ba5407bdd9.png">
I have restarted all the jobs and looks good.

Reply:
Actually, both the [public job](https://jenkins.public.confluent.io/job/cp-ansible/job/PR-894/4/) and [internal job](https://jenkins.confluent.io/job/confluentinc-pr/job/cp-ansible/job/PR-894/4/) were triggered at the same time here and the 2nd one failed with the standard 'community build disabled' message. 

It's the opposite, I believe. 'Public member' when enabled would make one's membership in organisation public. And  this information is what the public job uses. Since if we see the logs of that build, there's a part which states:
```
[GitHub Checks] contributor name is Clarence97
[GitHub Checks] publishNonConfluentIncPR is true
[GitHub Checks] publishConfluentIncPR is false
[GitHub Checks] inGithubOrg: false
```

FWIW, both jobs are testing the same thing (just in separate jenkins instances for internal and public builds).